src/Resources/config/services.yml

```diff
@@ -138,6 +138,24 @@ services:
         arguments:
             - "@contao.menu.matcher"
 
+    contao.menu.backend_menu_renderer:
+        class: Contao\CoreBundle\Menu\BackendMenuRenderer
+        arguments:
+            - "@twig"
+
+    contao.menu.backend_menu_builder:
+        class: Contao\CoreBundle\Menu\BackendMenuBuilder
+        arguments:
+            - "@knp_menu.factory"
+            - "@event_dispatcher"
+
+    contao.menu.backend_menu:
```

Comment:
According to discussion with @sheeep we don't need the menu as a service because of a custom renderer.